internal/command/init.go

```diff
@@ -49,6 +49,10 @@ type InitCommand struct {
 func (c *InitCommand) Run(args []string) int {
 	ctx := c.CommandContext()
 
+	// Initialization can be aborted by interruption signals
+	ctx, done := c.InterruptibleContext(ctx)
+	defer done()
+	
```

Comment:
This seems like something separate from what this PR was intended to address... or am I missing something? :grinning:

Reply:
You're right